### PR TITLE
Jawad

### PR DESCRIPTION
**Pull request by JawadHossain**
Add Login, Create Account Functionality 
Jira issue : https://cmput301.atlassian.net/browse/CMPUT-63

### CODE REVIEW

**Review by n00b4ever — APPROVED** [review]
looks good to me. should we make a user object class so it turns it into a User after pulling from database? for easier filling in MyProfile and ViewProfile

File: app/src/main/java/com/example/bookwormadventuresdeluxe2/LoginActivity.java [hunk 1]
```diff
@@ -89,4 +147,177 @@ public void onClick(View view)
...
+                                        case "ERROR_WRONG_PASSWORD":
+                                            EditTextValidator.wrongPassword(editTextPassword);
+//                                            progressBar.setVisibility(View.INVISIBLE);
+                                            break;
+
+                                        case "ERROR_INVALID_EMAIL":
+                                            /* Merged with next case as both set error to editTextEmail*/
+                                        case "ERROR_USER_NOT_FOUND":
+                                            EditTextValidator.emailNotFound(editTextEmail);
+//                                            progressBar.setVisibility(View.INVISIBLE);
```
Comment (huddy987) [inline app/src/main/java/com/example/bookwormadventuresdeluxe2/LoginActivity.java:286]: is there a reason for this?

File: app/src/main/java/com/example/bookwormadventuresdeluxe2/LoginActivity.java [hunk 2]
```diff
@@ -89,4 +147,177 @@ public void onClick(View view)
...
+                                {
+                                    /* Extract Firebase Error Code */
+                                    String errorCode = "";
+                                    errorCode = ((FirebaseAuthException) task.getException()).getErrorCode();
+
+                                    switch (errorCode)
+                                    {
+                                        case "ERROR_WRONG_PASSWORD":
+                                            EditTextValidator.wrongPassword(editTextPassword);
+//                                            progressBar.setVisibility(View.INVISIBLE);
```
Comment (huddy987) [inline app/src/main/java/com/example/bookwormadventuresdeluxe2/LoginActivity.java:279]: is there a reason for this?

**Review by huddy987 — CHANGES_REQUESTED** [review]

File: app/src/main/java/com/example/bookwormadventuresdeluxe2/LoginActivity.java (hunk 2 above)
Comment (JawadHossain) [reply app/src/main/java/com/example/bookwormadventuresdeluxe2/LoginActivity.java:279]: oh forgot to remove this. thanks for noticing

**Review by huddy987 — APPROVED** [review]

**Pull request merged**